format/format.go

```diff
@@ -235,12 +235,52 @@ func (p *Printer) applySpecialIndentRules(node *parse.ListNode) {
 			p.applySpecialLet(node.Nodes)
 		case IndentLetfn:
 			p.applySpecialLetfn(node.Nodes)
+		case IndentFor:
+			p.applySpecialFor(node.Nodes)
 		case IndentDeftype:
 			p.applySpecialDeftype(node.Nodes)
 		}
 	}
 }
 
+func (p *Printer) applySpecialForLet(nodes []parse.Node) {
+	i := -1
+	seenLet := false
```

Comment:
`seenLet` seems like the wrong name, since the meaning is actually "was the previous node a `let`". Maybe `prevLet` or something like that?